_posts/2021-05-12-244.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Cockpit 244
+author: kkoukiou
+date: 2021-05-12
+tags: cockpit linux
+slug: cockpit-244
+category: release
+summary: Cockpit with various improvements
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 244 and Cockpit-Machines 244.
+
+## Shell: No opportunistic sudo invocation
+
+Cockpit will no longer try to automatically run sudo on the very first login: you'll have to
+explicitly turn on administrative access once for each browser profile that you
+use to access Cockpit. This reduces a lot of noise about authentication failures in the journal.
+
+In addition, Cockpit now shows sudo's exact messages in the "Administrative access" dialog.
+
+![Sudo error messages](/images/sudo-error-messages.png)
+
+## Machines: Edit the MAC address of a VM's network interface
+
+To edit the MAC address of a virtual machine click the `Edit` button on the row
+of the network interface. A common scenario where this is needed is when
+configuring the DHCP server to map a static IP address to a certain MAC address.
+
+![VM edit MAC address](/images/vm-edit-mac-address.png)
```

Comment:
I didn't see this PR fly by. (I don't remember seeing it when I checked "release notes" tags on Monday...)

Sadly, the screenshot clearly shows improper alignment of the MAC address label. :cry: 

![Screenshot from 2021-05-12 09-29-29](https://user-images.githubusercontent.com/10246/117935993-9ec6c580-b304-11eb-93ab-c4cb1d735027.png)

This should be fixed... in a follow-up, I guess...

Reply:
(The two strings should be aligned to the baseline. Sure, the other form elements still have a bug where it's off by 1 or 2 pixels, based on rounding.... but that's an upstream PF issue. But this is _wildly_ off.)

Reply:
OH it's because the PR had another screenshot which was obsolete, so I took a new one. 
Let me check if I can quickly fix it.

Reply:
https://github.com/cockpit-project/cockpit-machines/pull/158

Reply:
Here's a new screenshot, cropped and optimized:
![nic-edit](https://user-images.githubusercontent.com/10246/117946412-945df900-b30f-11eb-9e3d-dd0c61a4c915.png)

Reply:
(Unless you'd rather have me fix up the previous one. This one has fake MAC addresses though.)

Reply:
http://sqa.fyicenter.com/1000208_MAC_Address_Validator.html#Result
This mac address is not valid.

The screenshot I have is already cropped. How do you optimize it? Can I do it?